pages/base.js

```diff
@@ -31,13 +31,17 @@ module.exports = {
     I.click(this.checkoutButton);
   },
 
-  async clearCart() {
+  async checkCartProducts() {
     I.click(this.cartSpoiler);
-    I.click(this.viewCartButton);
-    while (await I.grabNumberOfVisibleElements(this.deleteProductFromCartButton)) {
-      I.click(this.deleteProductFromCartButton);
-    };
+    let CartProducts = await tryTo(() => I.seeElement(this.deleteProductFromCartButton));
+    return CartProducts;
+  },
+  
+  async clearCart() {
```

Comment:
does not clear cart, if multiple products are in cart, only removes 1st product. Try to add `[last()]` to locator or use `for` cycle.